libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,24 +1,42 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
-  <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+  <div class="reading-list-item" [ngClass]="{'finished-item': b.finished}" *ngFor="let b of readingList$ | async">
     <div>
       <img class="reading-list-item--cover" [src]="b.coverUrl" />
     </div>
     <div class="reading-list-item--details">
       <strong class="reading-list-item--details--title">{{ b.title }}</strong>
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
+        <div *ngIf="b.finished" data-testing="finished-book-info" class="finished-book">
+            Finished book on {{ (b.finishedDate) | date: 'shortDate'}}
+        </div>
       </div>
     </div>
     <div>
-      <button
-        mat-icon-button
-        color="warn"
-        aria-label="Remove item from list"
-        [attr.aria-label]="'Remove ' + b.title + ' from reading list'"
-        (click)="removeFromReadingList(b)"
-      >
-        <mat-icon>remove_circle</mat-icon>
-      </button>
+
+      <div class ="reading-list-buttons">
+
+        <mat-checkbox
```

Comment:
Please use better UI elements, checkbox allows to select multiple values, which may cause issue in functionality.